crates/driver/src/infra/config/file/mod.rs

```diff
@@ -167,10 +162,6 @@ fn default_retry_interval() -> Duration {
     Duration::from_secs(2)
 }
 
-fn default_max_confirm_time() -> Duration {
-    Duration::from_secs(120)
```

Comment:
I am surprised this was set to 2 minutes?

Reply:
Yeah, the defaults in the driver are still a bit rough. Although in practice we'll likely never hit the full 2 minutes because the proposed solution would start to revert much earlier which also leads the driver to abort the submission.